models/Project.js

```diff
@@ -15,7 +15,7 @@ const ProjectSchema = new mongoose.Schema(
   {
     secret_key: { type: SecretKeySchema, required: true },
     owner: {
-      type: mongoose.Schema.Types.ObjectId,
+      type: String,
```

Comment:
여기 `String`으로 바꾸신 이유가 mongooseId가 아닌 user의 providerId를 저장해주기 위함인가요?

Reply:
넵 맞아요 프로젝트 삭제나 수정할때 owner를 알수있는 id가 필요한데 req.user 에 들어오는 토큰정보를 활용하고자 providerId로 바꿔주었습니다 아니면 
```suggestion
      owner: {
        _id: mongoose.ObjectId,
        providerId: 'abcdefg'
      }
```
이런식으로 두가지다 보관할까요? _id가 필요할까요?

Reply:
providerId로 찾는다면 _id는 필요없을 것 같습니다!
대신 성능개선을 위해 providerId를 index로 만들면 어떨까요?